test/FormGroupSpec.js

```diff
@@ -46,6 +46,15 @@ describe('FormGroup', function() {
     assert.ok(ReactTestUtils.findRenderedDOMComponentWithClass(instance, 'form-group-lg'));
   });
 
+  // This test case must come first, since the error only gets logged once.
```

Comment:
I really wish React had a way to turn that off, @aabenoja got pinned down on that for a while when working on the `ButtonInput` PR.